tcms/testcases/templates/testcases/mutable.html

```diff
@@ -233,15 +259,16 @@
         </form>
     </div>
 
+
 <script src="{% static 'bootstrap-select/dist/js/bootstrap-select.min.js' %}"></script>
 <script src="{% static 'bootstrap-switch/dist/js/bootstrap-switch.min.js' %}"></script>
-<script src="{% static "grappelli/jquery/jquery.min.js" %}"></script>
-<script src="{% static "grappelli/js/grappelli.min.js" %}"></script>
-<script src="{% static "admin/js/admin/RelatedObjectLookups.js" %}"></script>
+<script src="{% static 'grappelli/jquery/jquery.min.js' %}"></script>
+<script src="{% static 'admin/js/admin/RelatedObjectLookups.js' %}"></script>
 
+<script src="{% static 'bootstrap-duration-picker/dist/bootstrap-duration-picker.js' %}"></script>
+<script src="{% static 'bootstrap-duration-picker/dist/bootstrap-duration-picker-debug.js' %}"></script>
+<script src="{% static 'testcases/js/mutable.js' %}"></script>
```

Comment:
Order in which the files were listed in the HTML page is important. mutable.js needs functions defined in utils.js and jsonrpc.js. Move this line like it was before and don't change it.